### PR TITLE
Tweak chemical dispenser fill and nuke non-cm13 chemicals

### LINKED ISSUE #2438: Chemical dispensers should have cm chemicals only

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Tweaks the chemical dispenser's fill by removing and adding elements to match CM13. There are other small changes, like adding several new jugs and changing the colour of some chemicals.

Resolves #2438.

### CODE REVIEW

File: Resources/Prototypes/Catalog/ReagentDispensers/chemical.yml
Comment (Tunguso4ka) [inline Resources/Prototypes/Catalog/ReagentDispensers/chemical.yml:1]: thats upstream chem dispenser.

Comment (UbaserB) [reply Resources/Prototypes/Catalog/ReagentDispensers/chemical.yml:1]: fixed

File: Resources/Prototypes/_RMC14/Catalog/ReagentDispensers/chemical.yml
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Catalog/ReagentDispensers/chemical.yml:1]: Im already added both ChemDispenser Standard and Empty Inventories in `Resources/Prototypes/_RMC14/Entities/Structures/Machines/Science/Dispensers/chem.yml`

Comment (UbaserB) [reply Resources/Prototypes/_RMC14/Catalog/ReagentDispensers/chemical.yml:1]: fixed

File: Resources/Prototypes/_RMC14/Entities/Objects/Specific/chemical-containers.yml [hunk 1]
```diff
@@ -0,0 +1,452 @@
+- type: entity
+  name: jug
+  parent: BaseItem
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Entities/Objects/Specific/chemical-containers.yml:3]: you can use CMBeakerBase as parent here. Its located in `Resources/Prototypes/_RMC14/Entities/Objects/Medical/beaker.yml`
would be good if you moved your .yml file there too.

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Science/Dispensers/chem.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 - type: entity
   parent: ChemDispenser
-  id: CMDispenserChem
+  id: RMCDispenserChem
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Science/Dispensers/chem.yml:3]: add that to `migration.yml`

File: Resources/Prototypes/_RMC14/Entities/Structures/Machines/Science/Dispensers/chem.yml [hunk 3]
```diff
@@ -23,42 +23,45 @@
...
-  # water
-  - JugEthanol
-  - JugSugar
-  # sulfuric acid
-
 - type: entity
-  parent: CMDispenserChem
-  id: CMDispenserChemEmpty
+  parent: RMCDispenserChem
+  id: RMCDispenserChemEmpty
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Entities/Structures/Machines/Science/Dispensers/chem.yml:35]: `migration.yml`

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 4]
```diff
@@ -0,0 +1,499 @@
+- type: reagent
+  id: RMCAluminium
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:2]: idk why, but in cm-ss13 it called "Aluminum"
```
/datum/reagent/aluminum
name = "Aluminum"
```
from 'code/modules/reagents/chemistry_reagents/other.dm' line `461`

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+  name: reagent-name-aluminium
+  group: Elements
+  desc: reagent-desc-aluminium
+  physicalDesc: reagent-physical-desc-metallic
+  flavor: metallic
+  color: "#848789"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:8]: `#A8A8A8`

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 6]
```diff
@@ -0,0 +1,499 @@
...
+  meltingPoint: 660.0
+
+- type: reagent
+  id: RMCCarbon
+  name: reagent-name-carbon
+  group: Elements
+  desc: reagent-desc-carbon
+  physicalDesc: reagent-physical-desc-crystalline
+  flavor: bitter
+  color: "#22282b"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:19]:
```suggestion
  color: "#1C1300"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 7]
```diff
@@ -0,0 +1,499 @@
...
+  meltingPoint: 3550.0
+
+- type: reagent
+  id: RMCChlorine
+  name: reagent-name-chlorine
+  group: Elements
+  desc: reagent-desc-chlorine
+  physicalDesc: reagent-physical-desc-gaseous
+  flavor: bitter
+  color: "#a2ff00"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:30]:
```suggestion
  color: "#808080"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 8]
```diff
@@ -0,0 +1,499 @@
...
+            Poison: 2
+
+- type: reagent
+  id: RMCCopper
+  name: reagent-name-copper
+  group: Elements
+  desc: reagent-desc-copper
+  physicalDesc: reagent-physical-desc-metallic
+  flavor: metallic
+  color: "#b05b3c"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:57]:
```suggestion
  color: "#6E3B08"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 9]
```diff
@@ -0,0 +1,499 @@
...
+        amount: 0.4
+
+- type: reagent
+  id: RMCFluorine
+  name: reagent-name-fluorine
+  group: Elements
+  desc: reagent-desc-fluorine
+  physicalDesc: reagent-physical-desc-gaseous
+  flavor: bitter
+  color: "#ba85cc"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:88]:
```suggestion
  color: "#808080"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 10]
```diff
@@ -0,0 +1,499 @@
...
+  meltingPoint: 1064.76
+
+- type: reagent
+  id: RMCHydrogen
+  name: reagent-name-hydrogen
+  group: Elements
+  desc: reagent-desc-hydrogen
+  physicalDesc: reagent-physical-desc-gaseous
+  flavor: bitter
+  color: "#cccccc"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:126]:
```suggestion
  color: "#808080"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 11]
```diff
@@ -0,0 +1,499 @@
...
+  meltingPoint: -259.2
+
+- type: reagent
+  id: RMCIron
+  name: reagent-name-iron
+  group: Elements
+  desc: reagent-desc-iron
+  physicalDesc: reagent-physical-desc-metallic
+  flavor: metallic
+  color: "#434b4d"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:137]:
```suggestion
  color: "#C8A5DC"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 12]
```diff
@@ -0,0 +1,499 @@
...
+        amount: 0.4
+
+- type: reagent
+  id: RMCLithium
+  name: reagent-name-lithium
+  group: Elements
+  desc: reagent-desc-lithium
+  physicalDesc: reagent-physical-desc-shiny
+  flavor: metallic
+  color: "#d6c0a7"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:168]:
```suggestion
  color: "#808080"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 13]
```diff
@@ -0,0 +1,499 @@
...
+        probability: 0.05
+
+- type: reagent
+  id: RMCMercury
+  name: reagent-name-mercury
+  group: Elements
+  desc: reagent-desc-mercury
+  physicalDesc: reagent-physical-desc-shiny
+  flavor: metallic
+  color: "#929296"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:193]:
```suggestion
  color: "#484848"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 14]
```diff
@@ -0,0 +1,499 @@
...
+        component: ScrambledAccent
+
+- type: reagent
+  id: RMCPotassium
+  name: reagent-name-potassium
+  group: Elements
+  desc: reagent-desc-potassium
+  physicalDesc: reagent-physical-desc-shiny
+  flavor: metallic
+  color: "#c6c8cc"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:214]:
```suggestion
  color: "#A0A0A0"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 15]
```diff
@@ -0,0 +1,499 @@
...
+  boilingPoint: 759.0
+
+- type: reagent
+  id: RMCPhosphorus
+  name: reagent-name-phosphorus
+  group: Elements
+  desc: reagent-desc-phosphorus
+  physicalDesc: reagent-physical-desc-powdery
+  flavor: metallic
+  color: "#f5da9a"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:225]:
```suggestion
  color: "#832828"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 16]
```diff
@@ -0,0 +1,499 @@
...
+
+- type: reagent
+  id: RMCRadium
+  name: reagent-name-radium
+  group: Elements
+  parent: Uranium
+  desc: reagent-desc-radium
+  physicalDesc: reagent-physical-desc-glowing
+  flavor: metallic
+  color: "#00ff04"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:244]:
```suggestion
  color: "#C7C7C7"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 17]
```diff
@@ -0,0 +1,499 @@
...
+  boilingPoint: 1737.0
+
+- type: reagent
+  id: RMCSilicon
+  name: reagent-name-silicon
+  group: Elements
+  desc: reagent-desc-silicon
+  physicalDesc: reagent-physical-desc-crystalline
+  flavor: metallic
+  color: "#364266"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:255]:
```suggestion
  color: "#A8A8A8"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 18]
```diff
@@ -0,0 +1,499 @@
...
+  meltingPoint: 960.5
+
+- type: reagent
+  id: RMCSulfur
+  name: reagent-name-sulfur
+  group: Elements
+  desc: reagent-desc-sulfur
+  physicalDesc: reagent-physical-desc-powdery
+  flavor: bitter
+  color: "#fff385"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:284]:
```suggestion
  color: "#BF8C00"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 19]
```diff
@@ -0,0 +1,499 @@
...
+            Caustic: 0.1
+
+- type: reagent
+  id: RMCSodium
+  name: reagent-name-sodium
+  group: Elements
+  desc: reagent-desc-sodium
+  physicalDesc: reagent-physical-desc-metallic
+  flavor: bitter
+  color: "#a7b3d6"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:302]:
```suggestion
  color: "#808080"
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 20]
```diff
@@ -0,0 +1,499 @@
...
+  boilingPoint: 883.0
+
+- type: reagent
+  id: RMCUranium
+  name: reagent-name-uranium
+  group: Elements
+  desc: reagent-desc-uranium
+  physicalDesc: reagent-physical-desc-metallic
+  flavor: metallic
+  color: "#8fa191"
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:313]:
```suggestion
  color: "#B8B8C0"
```

**Review by Tunguso4ka — CHANGES_REQUESTED** [review]
colors and migration.yml

File: Resources/Locale/_RMC14/reagents/meta/elements.ftl [hunk 21]
```diff
@@ -0,0 +1,2 @@
+reagent-name-sulphuric-rmc-acid = sulphuric acid
+reagent-desc-sulphuric-rmc-acid = A corrosive chemical. Keep away from your face.
```
Comment (Tunguso4ka) [inline Resources/Locale/_RMC14/reagents/meta/elements.ftl:2]:
```suggestion
reagent-name-rmc-sulphuric-acid = sulphuric acid
reagent-desc-rmc-sulphuric-acid = A corrosive chemical. Keep away from your face.
```

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 22]
```diff
@@ -0,0 +1,499 @@
...
+    Drink:
+      effects:
+      - !type:SatiateThirst
+        factor: 4
+
+- type: reagent
+  id: RMCSulphuricAcid
+  name: reagent-name-sulphuric-rmc-acid
+  group: Toxins
+  desc: reagent-desc-sulphuric-rmc-acid
```
Comment (Tunguso4ka) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:457]:
```suggestion
  name: reagent-name-rmc-sulphuric-acid
  group: Toxins
  desc: reagent-desc-rmc-sulphuric-acid
```

**Review by Tunguso4ka — CHANGES_REQUESTED** [review]

File: Resources/Locale/_RMC14/reagents/meta/elements.ftl (hunk 21 above)
Comment (Tunguso4ka) [reply Resources/Locale/_RMC14/reagents/meta/elements.ftl:2]: WAIT, why is that file in `Resources/Locale/_RMC14/` ??? move it to `Resources/Locale/en-US/_RMC14`

**Review by Tunguso4ka — APPROVED** [review]
good work!

**Comment by UbaserB** [comment]
It's done now, I dont want to increase the scope too far.

File: Resources/Prototypes/_RMC14/Reagents/elements.yml [hunk 23]
```diff
@@ -0,0 +1,500 @@
+- type: reagent
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_RMC14/Reagents/elements.yml:1]:
```suggestion
# todo rmc14 these should have their effects checked for 13 values
- type: reagent
```

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]
We'll probably need to redo the chem machine later but this is mostly reagents for recipes anyway

**Review by QuietlyWhisper — APPROVED** [review]

**Pull request merged**